evennia/contrib/pathfinding/pathfinder.py

```diff
@@ -0,0 +1,212 @@
+from evennia import DefaultRoom as Room
```

Comment:
Please add a short module docstring giving and overview of  the functionality in this module.